src/client/theme-default/components/VPSocialLink.vue

```diff
@@ -10,20 +10,38 @@ const props = defineProps<{
 }>()
 
 const svg = computed(() => {
-  if (typeof props.icon === 'object') return props.icon.svg
-  return icons[props.icon]
+  if (typeof props.icon === 'object') {
+    return 'id' in props.icon
```

Comment:
this is wrong, we need to change the type in types/shared, we need to use built-in ones (sprite-social-icons folder) or use the VPSVGSpriteIcon (the symbol can be changed): the plugin should import and register each sprite symbol using `VPSVGSpriteIcon` in `socialIcons` map